examples/todo-mvc-svelte/README.md

```diff
@@ -0,0 +1,13 @@
+# Todo MVC example
```

Comment:
Some examples have no Readme, some of them have. Do you want to keep or remove it in this case?

Reply:
You can keep this; eventually they will have a README